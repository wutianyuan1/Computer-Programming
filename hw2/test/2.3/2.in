12
5
5
12
5
5
17
8
10
6
6
6
9
16
10
6
8
8
6
1
10
7
12
12
7
2
7
5
10
6
5
13
7
1
6
6
5
6
4
-1
9
6
3
16
8
10
10
5
-1
