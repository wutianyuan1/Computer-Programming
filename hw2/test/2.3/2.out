Please input the set A:
(resize) from 5 to 10
(resize) from 10 to 20
(resize) from 20 to 40
Please input the set B:
(resize) from 5 to 10
---Result---
12 5 5 17 8 10 6 6